docs/manuals/source/conf.py

```diff
@@ -302,51 +302,51 @@
 
 linkcheck_allowed_redirects = {
     # All HTTP redirections from the source URI to the canonical URI will be treated as "working".
-    r'https://bareos\.org/.*': r'https://bareos\.com/.*',
-    r'http://www\.bareos\.org': r'https://www\.bareos\.com/.*',
-    r'http://www\.bareos\.org/.*': r'https://www\.bareos\.com/.*',
-    r'https://www\.bareos\.org/.*': r'https://www\.bareos\.com/.*',
-    r'https://www\.bareos\.com/whitepapers/.*': r'https://www\.bareos\.com/.*',
-    r'https://bugs\.bareos\.org/.*': r'https://bugs\.bareos\.org/my_view_page\.php',
-    r'https://download\.bareos\.org': r'https://download\.bareos\.org/bareos',
-    r'https://download\.bareos\.com': r'https://download\.bareos\.com/bareos',
+    r"https://bareos\.org/.*": r"https://bareos\.com/.*",
+    r"http://www\.bareos\.org": r"https://www\.bareos\.com/.*",
+    r"http://www\.bareos\.org/.*": r"https://www\.bareos\.com/.*",
+    r"https://www\.bareos\.org/.*": r"https://www\.bareos\.com/.*",
+    r"https://www\.bareos\.com/whitepapers/.*": r"https://www\.bareos\.com/.*",
+    r"https://bugs\.bareos\.org/.*": r"https://bugs\.bareos\.org/my_view_page\.php",
+    r"https://download\.bareos\.org": r"https://download\.bareos\.org/bareos",
+    r"https://download\.bareos\.com": r"https://download\.bareos\.com/bareos",
 }
 linkcheck_request_headers = {
-#    "https://www.sphinx-doc.org/": {
-#        "Accept": "text/html",
-#        "Accept-Encoding": "utf-8",
-#    },
+    #    "https://www.sphinx-doc.org/": {
+    #        "Accept": "text/html",
+    #        "Accept-Encoding": "utf-8",
+    #    },
     "*": {
         "Accept": "text/html,application/xhtml+xml",
     }
 }
-linkcheck_retries = 2   # default 1
+linkcheck_retries = 2  # default 1
 linkcheck_timeout = 10  # default 300
```

Comment:
```suggestion
linkcheck_timeout = 10 # default 300
```

Reply:
This is from `bareos-check-sources`.

Reply:
ok same as before we let it like that then.